session/src/main/java/org/apache/iotdb/session/Session.java

```diff
@@ -1918,6 +1919,34 @@ public void createSchemaTemplate(Template template)
     defaultSessionConnection.createSchemaTemplate(req);
   }
 
+  /**
+   * Create a template with flat measurements, not tree structured. Need to specify datatype,
+   * encoding and compressor of each measurement, and alignment of these measurements at once.
+   */
+  public void createSchemaTemplate(
```

Comment:
We should explain the parameter of this method in java doc version.

Reply:
thanks! now there is a more detailed doc for the interface